src/admin-views/tribe-options-general.php

```diff
@@ -265,11 +265,40 @@
 ];
 
 if ( tribe_events_views_v2_is_enabled() ) {
+	// Push the control to the Filters tab
 	add_filter( 'tribe-event-filters-settings-fields', function ( $fields ) use ( $filter_activation ) {
 		$fields += $filter_activation;
 		return $fields;
 	} );
 } else {
+	/**
+	 * Filters the text for the "automatic" option.
+	 *
+	 * @since TBD
+	 *
+	 * @param string the displayed text.
+	 */
+	$automatic_text = apply_filters(
+		'tribe_events_liveupdate_automatic_label_text',
+		__( 'Enabled: datepicker selections automatically update calendar views.', 'the-events-calendar')
+	);
+	/**
+	 * Filters the text for the "manual" option.
+	 *
+	 * @since TBD
+	 *
+	 * @param string the displayed text.
+	 */
+	$manual_text    = apply_filters(
+		'tribe_events_liveupdate_manual_label_text',
+		__( 'Disabled: users must click Find Events to search by date.', 'the-events-calendar')
```

Comment:
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 43e20df4bb8620d1dacc9f314433e5ba</sub>